CHANGELOG.md

```diff
@@ -1,6 +1,59 @@
 This is the changelog for the Flux daemon; the changelog for the Helm
 operator is in [./CHANGELOG-helmop.md](./CHANGELOG-helmop.md).
 
+## 1.10.0 (2019-02-07)
+
+This release adds a `--registry-exclude-image` flag to the daemon to
+exclude certain (or all) images from scanning using glob expressions,
+allows for registries with self-signed certificates and fixes multiple
+bugs.
+
+### Fixes
+
+- Bumped `justinbarrick/go-k8s-portforward` to `1.0.2` to correctly
+  handle multiple paths in the `KUBECONFIG` env variable
+  [weaveworks/flux#1658][#1658]
+- Improved handling of registry challenge requests (preventing memory
+  leaks) [weaveworks/flux#1672][#1672]
+- Altered merging strategy for image credentials, which previously
+  could lead to Flux trying to fetch image details with credentials
+  from a different workload [weaveworks/flux#1702][#1702]
+
+### Improvements
+
+- Allow (all) images to be excluded from scanning
+  [weaveworks/flux#1659][#1659]
+- `--registry-insecure-host` now first tries to skip TLS host
+  host verification before falling back to HTTP, allowing registries
+  with self-signed certificates [weaveworks/flux#1526][#1526]
+- Allow `HOME` env variable when invoking Git which allows for mounting
+  a config file under `$HOME/config/git` [weaveworks/flux#1644][#1644]
+- Several documentation improvements and clarifications
+  weaveworks/flux{[#1656], [#1675], [#1681]}
+- Removed last traces of `linting` [weaveworks/flux#1673][#1673]
+- Warn users about external changes in sync tag
+  [weaveworks/flux#1695][#1695]
+
+### Thanks
+
+Lots of thanks to @2opremio, @alanjcastonguay, @bheesham, @brantb,
+@dananichev, @dholbach, @dmarkey, @hiddeco, @ncabatoff, @rade,
+@squaremo, @switchboardOp, @stefanprodan and @Timer for their
+contributions to this release, including those who I may have missed
```

Comment:
"including those" suggests you're clarifying a general statement -- for example "thanks to everyone who contributed, including those who logged issues", so it reads as slightly odd when you've already been specific.

Perhaps use "and anyone I've missed while writing this note."